RadixWallet/Features/InteractionReview/Sections/Sections+ExecutionSummary.swift

```diff
@@ -273,6 +273,8 @@ extension InteractionReview.Sections {
 				accountDepositSetting: accountDepositSetting,
 				accountDepositExceptions: accountDepositExceptions
 			)
+		case .some(.deleteAccounts(accountAddresses: let accountAddresses)):
+			fatalError()
```

Comment:
will go away when https://github.com/radixdlt/babylon-wallet-ios/pull/1389 is merged and I rebase to `main`.